### DISCUSSION TITLE
Change color of placeholder in <Input>

### DISCUSSION THREAD

**Discussion by AndrewSmir** [chakra-ui/chakra-ui#3110] [Q&A] [answered]
Hi! Is it possible to change the colour of placeholder in <Input> component?

**Comment by chr-ge** [comment, marked as answer]
Yes, you can use the `_placeholder` pseudo:
```jsx
<Input
  color='black'
  _placeholder={{ color: 'red.500' }}
/>
```

**Reply by RBOSAMIYA9** [reply]
How can we set dynamic color here?

**Reply by bernardomontoya** [reply]
@RBOSAMIYA9 Yes you can, `_placeholder` accepts an object with either static (like the example above) or dynamic values

**Reply by voigtito** [reply]
Hi, how do i change the placeholder color for a variant style? I tried this in the extendTheme configs:

```
Input: {
  variants: {
    filled: (props) => ({
      _placeholder: {
        color: 'black.900'
      },
    }),
  }
}
```

**Reply by aaronmcadam** [reply]
Hi @voigtito. Input is a multipart component, so we have to set these values on the `field` key instead:

```
export const Input = {
  variants: {
    outline: (props: Dict) => ({
      field: {
        bg: mode('white', 'gray.800')(props),
        borderColor: 'gray.300',
        _placeholder: {
          color: 'red.500',
        },
      },
    }),
  },
};
```

Hope this helps!
